### ISSUE TITLE
Relative paths in tabPlugins.cpp includes?

### ISSUE THREAD

**Issue by jwhendy** [monocasual/giada#132]
Hello,

I'm trying to compile with `--enable-debug` in order to figure out why I keep getting segfaults when trying to record audio. In doing this, I get a make error:

```
src/gui/elems/config/tabPlugins.cpp:32:30: fatal error: ../../core/const.h: No such file or directory
 #include "../../core/const.h"
                              ^
compilation terminated.
```

After investigating, I think the paths are incorrect for this this file, `src/gui/elems/config/tabPlugins.cpp`; here's the include section:
```
#include "../../core/const.h"
#include "../../core/conf.h"
```

From `src/gui/elems/config/tabAudio.cpp` (same directory):
```
#include "../../../core/const.h"
#include "../../../core/conf.h"
```

It builds fine after changing the paths to includes in `src/core` and `src/utils`. I'll have a pull request in a sec. Let me know if this is on my end. The data suggests it's legit, but I know extremely little C++.

**Comment by monocasual** [comment]
Hi @jwhendy thanks for the code and the bug report. The crash on recording audio is fixed in version 0.14.0 (83fd726).

**Comment by jwhendy** [comment]
No problem! Fun to find something my limited knowledge can actually fix. Re-building now to check on #134.
